src/pydata_sphinx_theme/theme/pydata_sphinx_theme/_templates/navbar-nav.html

```diff
@@ -1,4 +1,9 @@
 <ul id="navbar-main-elements" class="navbar-nav">
+    <div id="theme-switch" class="nav-item">
```

Comment:
can you put this switcher code in its own `.html` template and in a folder called `_templates/components/theme-switcher.html`?

Reply:
I assume that you would like to use the `components` folder to align on this issue: https://github.com/pydata/pydata-sphinx-theme/issues/577. In this PR the `components` folder does not exist yet don't you think it would be easier to simply add it in `templates` and then in a second PR the components will be moved to the appropriate sub-folder?

Reply:
I created a `theme-switcher.html` file can you confirm that it is usable when building the lib documentation ?

Reply:
hmm - are you building the documentation with `nox`? Can you double check that it is being reinstalled from scratch? I think any template that is in `_templates` should be included in the distribution without extra config needed.

Reply:
I'm not building from nox (always my weird environment issue) but that's no problem, I set it as a `_template` and pushed and the test documentation build itself nicely so i'd say it worked.